### PR TITLE
XOnlineDash Improvements

### PR DESCRIPTION
**Pull request by LukeUsher**

### CODE REVIEW

File: src/CxbxKrnl/EmuKrnl.cpp [hunk 1]
```diff
@@ -356,8 +356,8 @@ XBSYSAPI EXPORTNUM(252) xboxkrnl::DWORD NTAPI xboxkrnl::PhyGetLinkState
 )
 {
 	LOG_FUNC_ONE_ARG(Mode);
-
-	return XNET_ETHERNET_LINK_ACTIVE | XNET_ETHERNET_LINK_100MBPS | XNET_ETHERNET_LINK_FULL_DUPLEX;
+	LOG_UNIMPLEMENTED();
+	return 0;
```
Comment (Sarkie) [inline src/CxbxKrnl/EmuKrnl.cpp]: or create a XNET_NO_CONNECTION ?

Comment (Sarkie) [reply src/CxbxKrnl/EmuKrnl.cpp]: @LukeUsher you get the notification? Should we have a new enum?

Comment (LukeUsher) [reply src/CxbxKrnl/EmuKrnl.cpp]: No connection is reported by having no flags set, so this is an acceptable solution, even if it's not the best.

Perhaps we should add XNET_ETHERNET_NOT_CONNECTED, just for readability sake

Comment (Sarkie) [reply src/CxbxKrnl/EmuKrnl.cpp]: yeah, it shows intent to me and keeps the code readable.

**Pull request merged**